test/build/__snapshots__/build.test.js.snap

```diff
@@ -1,420 +1,17 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`snowpack build base-url: __snowpack__/env.js 1`] = `"export default {\\"MODE\\":\\"production\\",\\"NODE_ENV\\":\\"production\\"};"`;
+exports[`snowpack build html-environment-variables: __snowpack__/env.js 1`] = `"export default {\\"SNOWPACK_PUBLIC_BUILD_TIME\\":\\"build-time-replacement-configured-in-package.json\\",\\"MODE\\":\\"production\\",\\"NODE_ENV\\":\\"production\\"};"`;
 
-exports[`snowpack build base-url: _dist_/index.js 1`] = `
```

Comment:
Why is this `/_dist_/index.js` file now missing from the snapshots?

Reply:
oopsie sorry, I think I rebuild the snapshot when only running the single test, I'm fixing it

Reply:
fixed now